backend/i18n/translator/Squidex.Translator/State/TranslationService.cs

```diff
@@ -124,6 +124,18 @@ public void Save()
 
             Save("__todos.json", translationsTodo);
             Save("__ignore.json", translationToIgnore);
+            SaveKeys();
+        }
+
+        public void SaveKeys()
+        {
+            TranslationKeys translationKeys = new TranslationKeys();
```

Comment:
Can you please use "var" and a new line?

Reply:
I changed it to var, but what is the benefit of using var instead of a 'Type'?

Reply:
Less code and consistency ;)